lib/sycamore/sycamore/query/planner.py

```diff
@@ -77,7 +78,7 @@
 
 
 def process_json_plan(
-    plan: typing.Union[str, Any], operators: Optional[List[Type[LogicalOperator]]] = None
+    plan: typing.Union[str, Any], operators: Optional[List[Type[LogicalOperator]]] = None, postProcess: bool = False
```

Comment:
nit: we're using `post_process` style naming, also can you default to True? I think we always want this now

Reply:
Remove postprocess as a flag to process_json_plan (which was added to handle unit tests for that function).